compliance_checker/runner.py

```diff
@@ -66,6 +66,9 @@ def run_checker(cls, ds_loc, checker_names, verbose, criteria,
 
             score_groups = cs.run(ds, [] if skip_checks is None else skip_checks,
                                 *checker_names)
+            # TODO: consider wrapping in a proper context manager instead
```

Comment:
👍 thats not a bad idea! Can you make it happen? Perhaps open an issue so we don't lose track of it.